toolchain/sem_ir/entity_with_params_base.h

```diff
@@ -95,21 +94,27 @@ struct EntityWithParamsBase {
   // A block containing the pattern insts for the parameter lists.
   InstBlockId pattern_block_id;
   // A block containing, for each implicit parameter, a reference to the
-  // instruction in the entity's declaration block that depends on all other
-  // pattern-match insts pertaining to that parameter.
-  InstBlockId implicit_param_refs_id;
-  // A block containing, for each implicit parameter, a reference to the
   // instruction in the entity's pattern block that depends on all other
   // pattern insts pertaining to that parameter.
   InstBlockId implicit_param_patterns_id;
   // A block containing, for each explicit parameter, a reference to the
-  // instruction in the entity's declaration block that depends on all other
-  // pattern-match insts pertaining to that parameter.
-  InstBlockId param_refs_id;
-  // A block containing, for each explicit parameter, a reference to the
   // instruction in the entity's pattern block that depends on all other
   // pattern insts pertaining to that parameter.
   InstBlockId param_patterns_id;
+  // If this entity is a function, this block consists of references to the
+  // `AnyParam` insts that represent the function's call parameters. The "call
+  // parameters" are the parameters corresponding to the arguments that are
+  // passed to a `Call` inst, so they include `self` (if applicable) and the
+  // return slot, but do not include compile-time parameters.
+  //
+  // The parameters appear in declaration order: `self` (if present), then the
+  // explicit runtime parameters, then the return slot (which is "declared" by
+  // the function's return type declaration). This is not populated on imported
```

Comment:
You note the things which are included twice here, had you considered removing one of the mentions?

Reply:
I thought about it, but I couldn't find a way to do it without creating other problems. When I'm defining "`Call` parameters" it seems important to give examples of how it differs from other meanings of "parameters", and at some point I need to specify the parameter order fairly precisely since all the code that uses this field needs to agree on that, but combining the two forces me to front-load the ordering specification, which seems like it's dumping too much information on the reader all at once.

The `self` example in the first paragraph wasn't adding much, so I've at least removed that, but I'm not sure what else to do.